test/logger-helper-test.js

```diff
@@ -1011,8 +1011,8 @@ describe('logger-helpers tests', function () {
                     body: 'body',
                     test1: 'test2'
                 };
-                response.headers = undefined;
-                response._headers = undefined;
+                response.headers = {};
+                response._headers = {};
```

Comment:
I would prefer not to interact with _headers at all, being an internal detail.

Reply:
I agree with you. Will use public methods instead.